homeassistant/components/fitbit/sensor.py

```diff
@@ -231,28 +142,37 @@ def fitbit_configuration_callback(callback_data):
     )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
     """Set up the Fitbit sensor."""
     config_path = hass.config.path(FITBIT_CONFIG_FILE)
     if os.path.isfile(config_path):
-        config_file = load_json(config_path)
+        config_file: ConfigType = load_json(config_path)  # type: ignore[assignment]
```

Comment:
@KapJI maybe you have an idea how to annotate the return of `load_json()` :thinking:

Reply:
It's impossible to define correct type for arbitrary JSON: https://github.com/python/typing/issues/182
But you can define some approximation which will work in this case.

Reply:
without ignore, we get `error: Incompatible types in assignment (expression has type "Union[List[Any], Dict[Any, Any]]", variable has type "Dict[str, Any]")  [assignment]` ... after some testing and try&error I give up and simple marked it to ignore :see_no_evil:

Reply:
I think it's cleaner to use `cast` in this case.